src/browser/app/App.react.js

```diff
@@ -43,7 +44,7 @@ export default class App extends Component {
       ? `Something went wrong and we are very sorry about that. Drop us a message at info@scalac.io, thanks!`
       : `You'll hear from us very soon.`;
 
-    const message = `Hello, ${candidate.name}.`;
+    const message = candidate.name ? `Hello, ${candidate.name}.` : `Hello!`;
```

Comment:
This is a bit of a product question, but is this field used outside of this place? 
The thing is that "interrupt" the flow to get useless information

Reply:
Let me ask TT